### ISSUE TITLE
Atelier Elie GB Text is botched.

### ISSUE THREAD

**Issue by nanashi89** [m-mayday/game-translations#4]
![Elie no Atelier GB (Japan) (patched)-241102-190323](https://github.com/user-attachments/assets/7c2344ff-fe84-41ed-ac8a-31f0fc4d8725)
![Elie no Atelier GB (Japan) (patched)-241102-190332](https://github.com/user-attachments/assets/277570aa-f046-4948-ac82-62b6685ecf1c)
![Elie no Atelier GB (Japan) (patched)-241102-190404](https://github.com/user-attachments/assets/00a45a3d-a782-4ea0-a13f-8d6c856b2ca6)
![Elie no Atelier GB (Japan) (patched)-241102-190412](https://github.com/user-attachments/assets/98b7d338-a280-4db3-9807-2e28dc0be535)
These were taken using the mGBA core in RetroArch. Using the double-speed patch. It's not like this with the Atelier Marie GB Double Speed patch.  Can you remedy this, please? Thanks.

**Comment by m-mayday** [comment]
I just tried it and it looks fine for me, it seems like the patch wasn't applied correctly. In case it makes a difference, the patch is based off of the SGB enhanced version of the game.

That being said, and as specified in the readme, mGBA has a few other issues with these games, even the original ones, so it's not something I can fix. I recommend using a different core.

**Comment by nanashi89** [comment]
What core is reccomended?

**Comment by m-mayday** [comment]
Not sure as I don't use retroarch but any Game Boy Color specific should be fine. I did a quick test with Gearboy and encountered no issues.

If you still have issues with the text, then it's probably what I mentioned about the patch being based off of the SGB enhanced verison.

I'll close this as "Won't fix" but feel free to raise another issue if it still doesn't work.

**Comment by m-mayday** [comment]
@nanashi89 I made a mistake creating the patch I uploaded that would cause that botched text you see. ~~I reopened this while I wait on confirmation that the new patch works~~ Confirmed that it works, I'll leave this issue open for a while so others can see it. I sincerely apologize for this.

I still recommend not using mGBA btw, it makes certain textboxes disappear even in the original game.

**Comment by nanashi89** [comment]
![Elie no Atelier GB (Japan) (patched)-241103-001821](https://github.com/user-attachments/assets/31b1249c-979b-4bcf-8d7e-e118f898d979)
![Elie no Atelier GB (Japan) (patched)-241103-001903](https://github.com/user-attachments/assets/f631c905-bb17-4a1e-8a67-5b5aafad2d0c)
![Uploading Elie no Atelier GB (Japan) (patched)-241103-001907.png…]()
![Uploading Elie no Atelier GB (Japan) (patched)-241103-001916.png…]()


Ok! This is MUCH better! Thanks!
